### ISSUE TITLE
Duplicate exceptions from Ilogger + RequestTrackingModule

### ISSUE THREAD

**Issue by cijothomas** [Microsoft/ApplicationInsights-aspnetcore#861]
When an exception is thrown from Controller, RequestTrackingModule reports it.
https://github.com/Microsoft/ApplicationInsights-aspnetcore/blob/develop/src/Microsoft.ApplicationInsights.AspNetCore/DiagnosticListeners/Implementation/HostingDiagnosticListener.cs#L480

Since ILogger is also enabled by default now, it also captures this exception.

## Repro Steps
1. Use any WebAPI app, and throw unhandled exception from controller.

##  Actual Behavior
2 exception telemetry, 1 from Ilogger, 1 from request tracking module

## Expected Behavior
1

## Version Info 
SDK Version :
.NET Version :	 						
How Application was onboarded with SDK(VisualStudio/StatusMonitor/Azure Extension) :
OS :
Hosting Info (IIS/Azure WebApps/ etc) :

**Comment by cijothomas** [comment]
workaround
```
public void ConfigureServices(IServiceCollection services)
        {            
            services.ConfigureTelemetryModule<Microsoft.ApplicationInsights.AspNetCore.RequestTrackingTelemetryModule>((req, o) => req.CollectionOptions.TrackExceptions = false);
        }
```

**Comment by federicobarera** [comment]
as per this: https://github.com/Microsoft/ApplicationInsights-dotnet-logging/issues/268 please also keep in mind that other middlewares (without much config options) are possibly logging exceptions.

Can the workaround/configuration mentioned above please be advertised a bit better in the doccos?

Thanks

**Comment by alex-zyl** [comment]
Having similar issue when using Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.
ExceptionHandlerMiddleware itself logs unhandled exception and also writes "Microsoft.AspNetCore.Diagnostics.HandledException" event into diagnostic source which in turn triggers HostingDiagnosticListener to process the event.

**Comment by federicobarera** [comment]
@alex-zyl following the other thread, seems I have resolved with

1. Remove `UseApplicationInsights()` from `WebHost.CreateDefaultBuilder(args)`
2. In Startup.cs add: 
```
services.AddApplicationInsightsTelemetry();           services.ConfigureTelemetryModule<Microsoft.ApplicationInsights.AspNetCore.RequestTrackingTelemetryModule>((req, o) => req.CollectionOptions.TrackExceptions = false);
```
3. In ConfigureLogging:
```
c.AddApplicationInsights(h.Configuration["ApplicationInsights:InstrumentationKey"]);
```
4. Not being in `DeveloperMode` (I don't know if impacts or not)

I know they worked on adding logging directly into the main app insights package, but we are not using betas in prod so, our config is

```
<PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.6.1" />
 <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.9.1" />
```

**Comment by cijothomas** [comment]
@alex-zyl If you are using ExceptionHandlerMiddleware, you are right - you'll get duplicate logs. To prevent ai from getting it, apply a filter as shown below.
{
  "Logging": {
    "ApplicationInsights": {
      "LogLevel": {
        "Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware": "None"
      }
    },
    "LogLevel": {
      "Default": "Warning"
    }
  },
  "AllowedHosts": "*"
}

**Comment by cijothomas** [comment]
Closed with #863
